assets/js/details.js

```diff
@@ -4,7 +4,7 @@
   
   async function getMovieDetails () {
    const result = await fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}`);
-   const resultJson = await result.json();
+   const movie = await result.json();
```

Comment:
It's better now. But, You can destructre the properties from movie directly here.

Reply:
Done.